### PR TITLE
feat(scripts): Add spinnaker-diff.sh

### PR DESCRIPTION
**Pull request by AndesKrrrrrrrrrrr**
# ...

Attach a link to issue if relevant

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: scripts/spinnaker-diff.sh [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    podman run helm values --feature university-gateway --images "$images" --chart islandis --withMocks false >>"$f"
+  done
+}
+
+interact() {
+  echo "Now run 'nvim -Rd sugb-values.*.yaml'"
+  echo -n "Run now? [y/N] "
+  REPLY=n
+  if read -r -t 10 REPLY && [ "${REPLY,,}" = "y" ]; then
+    nvim -Rd "$OUT_DIR"/sugb-values.*.yaml
```
Comment (brynjarorng) [inline scripts/spinnaker-diff.sh:138]: Is it a good idea in a common script to use nvim?

File: scripts/spinnaker-diff.sh [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+    echo "Failed to fetching generated values from s3"
+  fi
+
+  for images in "star:*" "list:$IMAGES"; do
+    tag="${images/:*/}"
+    images="$OUT_DIR/${images/*:/}"
+    f="$OUT_DIR/sugb-values.local-$tag.yaml"
+    echo "Generating images for $tag to $f"
+    echo "feature-env-images: $images" >"$f"
+    podman run helm values --feature university-gateway --images "$images" --chart islandis --withMocks false >>"$f"
```
Comment (brynjarorng) [inline scripts/spinnaker-diff.sh:129]: Is it a good idea to use podman when the most wide spread app is docker?

**Comment by andes-it** [comment]
This PR's had a 15-day siesta. Lost its spark, did it? It's making an exit in 5 days unless the stale label disappears.

**Comment by andes-it** [comment]
This PR closed its doors. Sat around for a day after the stale tag, then reckoned it's not worth the fuss.

**Pull request closed without merging**